src/renderer/component/fileTile/view.jsx

```diff
@@ -101,8 +103,10 @@ class FileTile extends React.PureComponent<Props> {
                     label={__('Put something here!')}
                     onClick={e => {
                       // avoid navigating to /show from clicking on the section
-                      e.preventDefault();
-                      navigate('/publish');
+                      e.stopPropagation();
+                      // Strip prefix from the Uri and use that as navigation parameter
+                      const nameFromUri = uri.replace(/lbry:\/\//g, '').replace(/-/g, ' ');
+                      navigate('/publish', { name: nameFromUri });
```

Comment:
I'm hoping to avoid using navigation params for the publish section. Instead you can use something like:
```
doClearPublish() // to remove any existing publish data
doUpdatePublishForm({ name: nameFromUri }); // to populate the name
navigate('/publish')
```

Then nothing should need to be changed inside the `publishForm/view.jsx`